final-workflow.yml

```diff
@@ -76,16 +96,34 @@ deepPATH_bigwig: [ #the containing directory for bigwig files generated from the
 scaffold/bigwig
 ]
 #-------------------------------------------------------------------------------
-host: [ #host where Apollo2 lives
+#where apollo2 lives
+host_stage: [ 
 https://apollo-dev.nal.usda.gov
 ]
+host_production: [
+https://apollo-dev.nal.usda.gov
+]
+#authentication in apollo2
+login_apollo2_stage:  [
+user@ars.usda.gov,
+password
+]
+login_apollo2_production: [
+user@ars.usda.gov,
+password
+]
 #-------------------------------------------------------------------------------
-#authentication in apollo
+#Apollo server
 #username
-#password
-login_apollo2: [
-example@ars.usda.gov,
-ExamplePassword
-]
+Apollo_account: user@apollo-stage-node1.nal.usda.gov
+Gmod_account: user@gmod-stage-node1.nal.usda.gov
+Gmod_stage_account: user@gmod-dev.nal.usda.gov
 #-------------------------------------------------------------------------------
+#symlink PATH
+MAIN_PATH: /app/data/hsiukang.huang  
+LINK_DIR: apimel-(Apis_mellifera)/GCF_003254395.2
+ASSEMBLY_NAME: Amel_HAv3.1
```

Comment:
@HsiuKangHuang i don't think this is needed since this is already given in tree[1]

Reply:
Okay! I can remove this.